### PR TITLE
Add Part Of Speech missing data to Token

### PR DESCRIPTION
**Pull request by dvaldivia**
Hi,

This PR add the missing data returned by the API to the Token, before only the Tag was being returned, I added all the data to the existing PartOfSpeech object and replaced it's hydratation on the Token as well

### CODE REVIEW

**Comment by dvaldivia** [comment]
I signed it!

**Comment by dvaldivia** [comment]
@lukesneeringer you are welcome :) What's next? Do I need to do anything else?

File: language/google/cloud/language/syntax.py [hunk 1]
```diff
@@ -20,7 +20,45 @@
 
 
 class PartOfSpeech(object):
-    """Part of speech of a :class:`Token`."""
+    """A Google Cloud Natural Language API Part of speech object.
+    Rich data about the Part of Speech of a token returned by the API
```
Comment (dhermes) [inline language/google/cloud/language/syntax.py]: Can you add a blank line between these two. Motivation, from [PEP257][1]

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

[1]: https://www.python.org/dev/peps/pep-0257/

File: language/tests/unit/test_api_responses.py [hunk 2]
```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from copy import copy
 import unittest
+from copy import copy
```
Comment (dhermes) [inline language/tests/unit/test_api_responses.py]: This import should not have moved (alphabetical order)

**Comment by dvaldivia** [comment]
@dhermes did as you requested, let me know if I should change anything else.

**Pull request merged**